Providers/nxOMSPerfCounter/MSFT_nxOMSPerfCounterResource.schema.mof

```diff
@@ -8,10 +8,11 @@ class MSFT_nxOMSPerfCounterObject
     [write] string PerformanceCounter[];
 };
 
-[ClassVersion("1.1.0")]
+[ClassVersion("2.0.0")]
```

Comment:
ClassVersion should not be changed. Resource version is driven from module version only.